### PR TITLE
Assignement4 teammembers

### PR DESCRIPTION
**Pull request by soumyagangamwar**
Screenshot (9)

### CODE REVIEW

File: assignments/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+**Build your Team App Assignment**
```
Comment (ap221882) [inline assignments/README.md:1]: add .parcel-cache to .gitignore

File: assignments/src/App.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { createElement } from "react";
-import CHeader from "./CHeader";
+import CHeader from "./CHeader.js";
+import CardContainer from "./cards/CardContainer.js";
 const heading = React.createElement("h1", { id: "title" }, "Namaste react!!");
```
Comment (ap221882) [inline assignments/src/App.js:6]: have a blank line after global import and imports for better code readability

File: assignments/src/cards/CardContainer.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import teamData from "../assets/json/teamMembers.json";
+import CardComponent from "./Cards.js";
+import "./cards.css";
+
+const CardContainer = () => {
+  return (
+    <>
+      <div className="card-container">
```
Comment (ap221882) [inline assignments/src/cards/CardContainer.js:8]: You don't need Fragment here.

**Review by ap221882 — COMMENTED** [review]
Nicely coded. Added comments.

**Comment by soumyagangamwar** [comment]
Thank you for reviewing my code .Fixed review omment except gitignore parcel cache

**Pull request merged**